examples/seldon_deployment/run.py

```diff
@@ -0,0 +1,209 @@
+        --min-accuracy 0.80
+    """
+
+    if stop_service:
+        service = load_last_service_from_step(
+            pipeline_name="continuous_deployment_pipeline",
+            step_name="model_deployer",
+            running=True,
+        )
+        if service:
+            service.stop(timeout=100)
+        return
+
+    if model_flavor == "tensorflow":
+        seldon_implementation = "TENSORFLOW_SERVER"
+        trainer_config = TensorflowTrainerConfig(epochs=epochs, lr=lr)
+        trainer = tf_trainer(trainer_config)
+        evaluator = tf_evaluator()
+        predict_preprocessor = tf_predict_preprocessor()
+    else:
+        seldon_implementation = "SKLEARN_SERVER"
+        trainer_config = SklearnTrainerConfig(
+            solver=solver,
+            penalty=penalty,
+            C=penalty_strength,
+            tol=toleration,
+        )
+        trainer = sklearn_trainer(trainer_config)
+        evaluator = sklearn_evaluator()
+        predict_preprocessor = sklearn_predict_preprocessor()
+
+    # Initialize a continuous deployment pipeline run
+    deployment = continuous_deployment_pipeline(
+        importer=importer_mnist(),
+        normalizer=normalizer(),
+        trainer=trainer,
+        evaluator=evaluator,
+        deployment_trigger=deployment_trigger(
+            config=DeploymentTriggerConfig(
+                min_accuracy=min_accuracy,
+            )
+        ),
+        model_deployer=seldon_model_deployer(
+            config=SeldonDeployerConfig(
+                model_name="mnist",
+                step_name="model_deployer",
+                replicas=1,
+                implementation=seldon_implementation,
+                secret_name="seldon-init-container-secret",
+                kubernetes_context=kubernetes_context,
+                namespace=namespace,
+                base_url=base_url,
+                timeout=120,
+            )
+        ),
+    )
+
+    deployment.run()
+
+    # Initialize an inference pipeline run
+    inference = inference_pipeline(
+        dynamic_importer=dynamic_importer(),
+        predict_preprocessor=predict_preprocessor,
+        prediction_service_loader=prediction_service_loader(
+            SeldonDeploymentLoaderStepConfig(
+                pipeline_name="continuous_deployment_pipeline",
+                step_name="model_deployer",
+            )
+        ),
+        predictor=predictor(),
+    )
+
+    inference.run()
+
+    service = load_last_service_from_step(
+        pipeline_name="continuous_deployment_pipeline",
+        step_name="model_deployer",
+        running=True,
+    )
+    if service:
+        print(
+            f"The Seldon prediction server is running remotely as a Kubernetes "
+            f"service and accepts inference requests at:\n"
+            f"    {service.prediction_url}\n"
+            f"To stop the service, re-run the same command and supply the "
+            f"`--stop-service` argument."
+        )
```

Comment:
What would you think about separating deployment and inference into two functions that can be called separately from the cli. It makes it more interactive for the user and makes it a bit more obvious for the user

Reply:
This is a great idea. However, I spent too much time as it is on pimping up this example and there are tons of other things I need to do to finish up the integration (seldon core deployer component, secret management, maybe add in some endpoint health-checks). I would rather wave this idea for another time.

Reply:
Alright leave this as is for now, If I get the time I'll do it within the examples overhaul :)

Reply:
Scratch that, I went ahead and implemented it anyway :D